src/common/components/build-status/index.js

```diff
@@ -47,7 +39,7 @@ const BuildStatus = (props) => {
 
 BuildStatus.propTypes = {
   link: PropTypes.string,
-  status:  PropTypes.oneOf(['success', 'pending', 'error']),
+  colour: PropTypes.oneOf(['green', 'yellow', 'red', 'grey']),
```

Comment:
I've replaced the "success", "pending", "error" statuses with a straight map to colours. The spec states that grey and blue will be added, and so colours will not map directly to the old Vanilla status colours.